src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="./main.js">
+    </script>
+    <form class="block">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-filed">
+            Surname:
+          <input type="text" name="surname" autocomplete="of">
+        </label>
+
+        <label class="form-filed">
+            Name:
+          <input type="text" name="name"  autocomplete="off">
+        </label>
+
+        <label class="form-filed">
+            How old are You?
+          <input
+            autocomplete="off"
+            type="number" 
+            name="age" 
+            value="12"
+            min="1" 
+           max="100"
+          >
```

Comment:
```suggestion
          <input
            autocomplete="off"
            type="number" 
            name="age" 
            value="12"
            min="1" 
            max="100"
          >
```
Выравнивай атрибуты одинаково